clang/lib/CIR/CodeGen/CIRGenBuiltinX86.cpp

```diff
@@ -158,6 +163,33 @@ computeFullLaneShuffleMask(CIRGenFunction &cgf, const mlir::Value vec,
 
   outIndices.resize(numElts);
 }
+
+static mlir::Value emitPrefetch(CIRGenFunction &cgf, unsigned builtinID,
```

Comment:
```suggestion
static void emitPrefetch(CIRGenFunction &cgf, unsigned builtinID,
```
Given that this always returns `{}` it seems better for it to just be void.